linter/ginkgo_linter.go

```diff
@@ -1651,6 +1793,31 @@ func isExprErrFunc(pass *analysis.Pass, expr ast.Expr) bool {
 	return false
 }
 
+// Returns whether the expression is a function that takes in a single gomega.Gomega argument and
+// returns nothing, e.g. `func(g gomega.Gomega) { g.Expect(myStringMaker()).ToNot(BeEmpty()) }`
```

Comment:
The `Gomega` parameter is not necessary  a single parameter. The check should be about the first parameter, if exists.